src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -494,3 +494,9 @@ public net.minecraft.world.item.enchantment.Enchantment locationContext(Lnet/min
 public net.minecraft.world.item.enchantment.Enchantment entityContext(Lnet/minecraft/server/level/ServerLevel;ILnet/minecraft/world/entity/Entity;Lnet/minecraft/world/phys/Vec3;)Lnet/minecraft/world/level/storage/loot/LootContext;
 public net.minecraft.world.item.enchantment.Enchantment blockHitContext(Lnet/minecraft/server/level/ServerLevel;ILnet/minecraft/world/entity/Entity;Lnet/minecraft/world/phys/Vec3;Lnet/minecraft/world/level/block/state/BlockState;)Lnet/minecraft/world/level/storage/loot/LootContext;
 public net.minecraft.world.item.enchantment.Enchantment applyEffects(Ljava/util/List;Lnet/minecraft/world/level/storage/loot/LootContext;Ljava/util/function/Consumer;)V
+
+# HierarchicalModel animation methods
+public net.minecraft.client.model.HierarchicalModel animate(Lnet/minecraft/world/entity/AnimationState;Lnet/minecraft/client/animation/AnimationDefinition;F)V
+public net.minecraft.client.model.HierarchicalModel animateWalk(Lnet/minecraft/client/animation/AnimationDefinition;FFFF)V
+public net.minecraft.client.model.HierarchicalModel animate(Lnet/minecraft/world/entity/AnimationState;Lnet/minecraft/client/animation/AnimationDefinition;FF)V
+public net.minecraft.client.model.HierarchicalModel applyStatic(Lnet/minecraft/client/animation/AnimationDefinition;)V
```

Comment:
This will not work. The methods are protected and can be overridden if desired, which would break the access transformer.

Reply:
Hmm? No, that's not true. When everything is already compiled (dependencies, production), a protected method can override a public one without issue. At worst, this is a source breaking change _only_ if you're overriding those methods yourself, not even if a dependency does it.

Reply:
My apologies for the inaccuracy. But sure, anything can happen after compilation, and this would only affect source compilation if not updated. Still, I don't think we should be widening protected methods just because in this one case it happened to work out that no other methods in the original dependency didn't override it.